modules/network-observability-nodes-taints-tolerations.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// network_observability/network-observability-scheduling-resources.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="network-observability-multi-tenancy{context}"]
+= Using tolerations to control Network Observability deployment in specific nodes
+You can configure the `FlowCollector` to allow you to deploy Network Observability components in specific nodes. The `spec.agent.ebpf.advanced.scheduling`, `spec.processor.advanced.scheduling`, `spec.consolePlugin.advanced.scheduling` specifications have the following settings you can configure: 
+
+* `NodeSelector`
+* `Tolerations`
+* `Affinity`
+* `PriorityClassName`
+
+.Sample `FlowCollector` resource for `spec.<component>.advanced.scheduling` 
+[source,yaml]
+----
+apiVersion: flows.netobserv.io/v1beta2
+kind: FlowCollector
+metadata:
+  name: cluster
+spec:
+# ...
+advanced:
+   scheduling:
```

Comment:
think spacing is a little off here. For scheduling section

Reply:
Ok I adjusted this based on the sample Mohamed gave me.

Reply:
like the `s` in scheduling should be below the `v` in advanced. Thats the off part for me

Reply:
Okay I shifted everything over 1 space, which is what it took to move the `s` below the `v`

Reply:
Looks good!